rest/config.go

```diff
@@ -112,22 +112,23 @@ func (c ClusterConfig) CBGTEnabled() bool {
-	ChannelIndex       *ChannelIndexConfig            `json:"channel_index,omitempty"`        // Channel index settings
-	RevCacheSize       *uint32                        `json:"rev_cache_size,omitempty"`       // Maximum number of revisions to store in the revision cache
-	StartOffline       bool                           `json:"offline,omitempty"`              // start the DB in the offline state, defaults to false
-	Unsupported        *UnsupportedConfig             `json:"unsupported,omitempty"`          // Config for unsupported features
-	OIDCConfig         *auth.OIDCOptions              `json:"oidc,omitempty"`                 // Config properties for OpenID Connect authentication
+	Name               string                         `json:"name,omitempty"`                  // Database name in REST API (stored as key in JSON)
+	Sync               *string                        `json:"sync,omitempty"`                  // Sync function defines which users can see which data
+	Users              map[string]*db.PrincipalConfig `json:"users,omitempty"`                 // Initial user accounts
+	Roles              map[string]*db.PrincipalConfig `json:"roles,omitempty"`                 // Initial roles
+	RevsLimit          *uint32                        `json:"revs_limit,omitempty"`            // Max depth a document's revision tree can grow to
+	ImportDocs         interface{}                    `json:"import_docs,omitempty"`           // false, true, or "continuous"
+	Shadow             *ShadowConfig                  `json:"shadow,omitempty"`                // External bucket to shadow
+	EventHandlers      interface{}                    `json:"event_handlers,omitempty"`        // Event handlers (webhook)
+	FeedType           string                         `json:"feed_type,omitempty"`             // Feed type - "DCP" or "TAP"; defaults based on Couchbase server version
+	AllowEmptyPassword bool                           `json:"allow_empty_password,omitempty"`  // Allow empty passwords?  Defaults to false
+	CacheConfig        *CacheConfig                   `json:"cache,omitempty"`                 // Cache settings
+	ChannelIndex       *ChannelIndexConfig            `json:"channel_index,omitempty"`         // Channel index settings
+	RevCacheSize       *uint32                        `json:"rev_cache_size,omitempty"`        // Maximum number of revisions to store in the revision cache
+	StartOffline       bool                           `json:"offline,omitempty"`               // start the DB in the offline state, defaults to false
+	Unsupported        *UnsupportedConfig             `json:"unsupported,omitempty"`           // Config for unsupported features
+	OIDCConfig         *auth.OIDCOptions              `json:"oidc,omitempty"`                  // Config properties for OpenID Connect authentication
+	ViewQueryTimeoutMs *uint32                        `json:"view_query_timeout_ms,omitempty"` // The view query timeout in milliseconds
```

Comment:
Do you think it's required to set the view query timeout in milliseconds in the config?  It seems more user-friendly to use seconds for this kind of timeout.  Was this specifically for QE's benefit?